### ISSUE TITLE
No way to disable issues with comments

### ISSUE THREAD

**Issue by OmgImAlexis** [eslint/eslint#12548]
**Tell us about your environment**

* **ESLint Version:** N/A
* **Node Version:** v12.5.0
* **npm Version:** 6.13.0

**What parser (default, Babel-ESLint, etc.) are you using?**
Default with [`xo`](https://github.com/xojs/xo).
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
// eslint-disable-next-line capitalized-comments
/* spell-checker: disable */
```

```bash
xo --quiet
```

**What did you expect to happen?**
`capitalized-comments` to apply to the comment below it.

**What actually happened? Please include the actual, raw output from ESLint.**
```
  ✖  131:31  capitalized-comments rule is disabled but never reported.  eslint-comments/no-unused-disable
```

If I remove the `eslint-disable-next-line` line then I get this.
```
  ✖  131:31   Comments should not begin with a lowercase character.      capitalized-comments
```

**Are you willing to submit a pull request to fix this bug?**
Possibly?

**Comment by g-plane** [comment]
Thanks for this issue.

It seems that you've added an additional comment after `eslint-disable-next-line`. If you will want to keep that comment (about spell checker), you can consider adding `eslint-disable-line` at the same line which ESLint reports.

**Comment by OmgImAlexis** [comment]
The issue it has is the spell check comment though? That’s why I opened this. I had assumed the eslint comment would apply for the spelling comment but it doesn’t seem to work.

**Comment by g-plane** [comment]
Oh sorry. I didn't realize that.

**Comment by thewalla07** [comment]
Hi, this may not be related to ESLint directly, but rather with the [eslint-plugin-eslint-comments](https://github.com/mysticatea/eslint-plugin-eslint-comments) dependency used by [xo](https://github.com/xojs/xo). This dependency is the one that actually reports this error in xo

**Comment by thewalla07** [comment]
@OmgImAlexis can you give some more information on your environment and any configs you have for xo or ESLint in your project? I am struggling to reproduce this again

**Comment by OmgImAlexis** [comment]
I’m using a default install of xo.

I’m seeing this 100% of the time on my project so if there’s anything you need me to test just let me know.

**Comment by OmgImAlexis** [comment]
I noticed you talked about Linux and windows. I’m on macOS, doubt that’s makes a difference but figured I’d mention it.

**Comment by kaicataldo** [comment]
@OmgImAlexis Is there any chance you could make a minimal reproduction case repo? It's tough to know what's going on with all the moving parts at the moment.
